docs/12-experimental.md

```diff
@@ -0,0 +1,48 @@
+# Experimental features
```

Comment:
Suggestion: Move this so that it is the last option under the reference tab.

In the spirit of making our documentation as graspable as possible I think this should be moved into the reference tab as the last option.